app/assets/javascripts/fae/_tables.js

```diff
@@ -267,6 +267,8 @@ Fae.tables = {
 
       Fae.tables.sizeFixedHeader();
     });
+
+    $toggle.trigger('click');
```

Comment:
I did, but discovered that there is also static markup in admin views outputting "Open All" on page load. Looks to be spread across Fae installs and falls outside an easily modifiable helper method. Simulating a click seemed to be the lowest touch way to handle this ask, but I agree it feels a bit tacked on 🤔